### ISSUE TITLE
Tractor will not drive to combine

### ISSUE THREAD

**Issue by Flashsolutions** [Stephan-S/FS19_AutoDrive#1008]
AutoDrive Version: 1.0.7.0-33
Release or custom zip: custom
Map (if applicable): Eiserholt
Vehicle / type (applicable): MF 7726

**Describe the bug**
1. Tractor will not drive to combine either when it reaches pre-call value nor once the combine is full.  This typically only happens when I first try to start harvesting a field.  Once I manage to get it running, it behaves better.
See video at 6:35 and 7:50 and 9:05
2. AD could not find the Field waypoint which is obviously present and a path to it is shown and is found thereafter.

**If applicable - Steps to reproduce the behavior**

**Screenshots**
https://youtu.be/wUBClyTho2k
[log.txt](https://github.com/Stephan-S/FS19_AutoDrive/files/3945527/log.txt)

**Comment by Stephan-S** [comment]
You are doing so many things to make this Impossible for AD.
First of all: don't use Dual way roads so much. They we're never meant to be used that way and they cause a Lot of issues with the current dijkstra Implementation. This might Work better when the live Mode is done.
This is why he is having Trouble reaching your field destination. But the again, you could have also reversed a little more to make it easier for AD to find the best starting Point. It's obviously Not trivial when you have so many Points there at the crossing and especially with the Dual way roads that Point 180° in the other direction logically.

Thema when you finally Manager to get your tractor to the field point, you let him stand there pointing into the crop which you specifically Set to avoid in the settings.
Thema when you enable all debug Levels and have the Edit Mode enabled, you can even See what the AD Pathfinder does. And He Marks the next possible cells He can Drive to with a Red Line and stops Processing. So He stops since He cant find a Start avoiding the crop right in Front.

If you disable the Edit Mode, He would have restarted the calculation but since you didn't, He was stuck there.

Then when you restarted AD with the full combine He found a perfect way and even showed you this. You would only have to disable the Edit Mode. This is a debug function, so we developers or even you Users can see the Pathfinder process. To Continue, Just disable the Edit mode

**Comment by Flashsolutions** [comment]
Actually, it does this without debug mode turned on.  I duplicated the situation after first encountering it without debug mode.  I just wanted to make sure I documented it.  

I understand your logic regarding the tractor facing the crop, but the crop he is facing has already been harvested, so I did not see it the same way as you described.

I believe I did drive the tractor out onto the road and took another path into the field to the Field 53 waypoint which it found and again stopped and would not drive to the combine.

So, if I understand you correctly, I would need to pick a point outside the field as its starting point?  I will set that up and see if I can get that to work.

By the way, what is LIVE MODE?  

While I did not document it, once I got AD moving, there were times when departing the harvester that the tractor left the field (53) and drove across the road into an adjacent field before turning back, crossing the road and returning to field 53.

It also runs into trees toward the end of the harvest even though there is a path around the entire field.  These are things I have replicated time and time again.

I will write that up separately, but just wanted to note that in case you wanted to comment on it.

**Comment by Flashsolutions** [comment]
Okay, I did as you suggested and created a waypoint on the perimeter of the field and a one way path from the road leading into the path toward the new waypoint.

Started the harvester fresh and let it get half way full before starting the tractor on its way.  It proceeded to the new waypoint and stopped, waited till the pre call level reached 80% and drove to it, waited, and then when the combine was full, drove to it and unloaded.

The tractor then drove back to the new waypoint, turned off the engine and the combine continued till it reached 80%, but this time the tractor did not leave for the combine.  The combine continued till it reached 100% fill level and did not extend the pipe and tractor remains at its waypoint.

I did not have debug on nor did I have edit mode on.

Using the same settings as before for both tractor and combine.

**Comment by galdavant** [comment]
Having same issue. Tractor will go to combine and unload it fine on small fields but if it is a larger field the tractor seems to not be able to find the combine. It will just sit there. If I bring combine closer it detects it so I am assuming it is a distance related thing.

**Comment by TyKonKet** [comment]
**Hello, Thank you for opening this issue!**
Since we have just released [the new refactored version of AutoDrive](https://github.com/Stephan-S/FS19_AutoDrive/releases/tag/1.1.0.0), we can no longer provide any feedback or support for open bug reports related to the old code base.
If the problem persists, please reopen a new issue with all the required details.
Thank you for understanding and we hope you continue enjoying our mod 😄
